soh/src/code/z_parameter.c

```diff
@@ -5578,7 +5579,7 @@ void Interface_Draw(PlayState* play) {
             Interface_DrawAmmoCount(play, 3, interfaceCtx->cRightAlpha);
         }
 
-        if (CVarGetInteger(CVAR_SETTING("DpadEquips"), 0) != 0) {
+        if (CVarGetInteger(CVAR_SETTING("CVAR_ENHANCEMENT"), 0) != 0) {
```

Comment:
Replaced the wrong thing with CVAR_ENHANCEMENT here XD

Reply:
Fixed.